### DISCUSSION TITLE
Close notification

### DISCUSSION THREAD

**Discussion by yriveiro** [folke/noice.nvim#411] [Q&A] [answered]
Hi,

How do I close a Noice notification when it is shown? Sometimes the notification is too big and I need to close it but I don't know how to do it, is there any keymap to do that?

**Comment by folke** [comment]
Check the nvim-notify docs

**Comment by yriveiro** [comment, marked as answer]
Didn't find anything useful, I asked on `nvim-notify` if that is possible but I fear that I will need to build custom code to do that using some mimic from this [code](https://github.com/rcarriga/nvim-notify/blob/50d037041ada0895aeba4c0215cde6d11b7729c4/lua/notify/windows/init.lua#L111)

**Reply by folke** [reply]
It's in the nvim-notify docs

```lua
          require("notify").dismiss({ silent = true, pending = true })
```

**Reply by yriveiro** [reply]
I did the search use the word `close`, my bad, thanks for the help @folke
